_data/projects/RESTEasyCLI.yml

```diff
@@ -0,0 +1,12 @@
+name: RESTEasyCLI
+desc: Handy REST API client on your terminal
+site: https://github.com/rapidstack/RESTEasyCLI
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- python
+- cli
+- yml
+- REST
+upforgrabs:
+  name: help needed
+  link: https://github.com/rapidstack/RESTEasyCLI/issues
```

Comment:
This link should be to the `help needed` issues: `https://github.com/rapidstack/RESTEasyCLI/labels/help%20wanted`

Reply:
Oh yeah, I forgot. I will make changes. Thanks for the info.